### ISSUE TITLE
Installation with Bioconda

### ISSUE THREAD

**Issue by CharlesDMore** [barricklab/breseq#269]
Hello,

I installed breseq using bioconda following the website instructions. Everything seemed OK. However I am unable to use the "breseq" command during the test drive tutorial. Is there a specific location within miniconda3 where the test_drive directory should be created? I tried multiple paths without success.

Thank you very much!

**Comment by jeffreybarrick** [comment]
The `test_drive` directory can be anywhere on your machine.

Are you able to run `breseq` on its own or find it in your `$PATH` with these commands?

```
breseq
which breseq
```

If not, then maybe something is wrong with the installation or how you have miniconda3 set up in terms of the `$PATH`.

**Comment by CharlesDMore** [comment]
Indeed, I was unable to use both "breseq" and "which breseq". I did a fresh install. Now I can use the breseq command! However, I now an unable to access access R somehow through this command, as it returns this error: 

****bowtie2  :: version 2.2.5 [/home/admin/miniconda3/bin/bowtie2]
---> ERROR Could not determine version of installed executable "R".
---> For found executable installed at [/home/admin/miniconda3/bin/R]
---> Command "R --version" returned:
/home/admin/miniconda3/lib/R/bin/exec/R: error while loading shared libraries: libreadline.so.6: cannot open shared object file: No such file or directory ******

I do not know what this means. I also cant find libreadline.so.6 (I have libreadline.so.8)

Thank you again

**Comment by jeffreybarrick** [comment]
Hmm... There seems to be something wrong with your R install now (as in it is incomplete or doesn't work for your machine) You may need to try re-installing that in conda... which should be installing things that work.

**Comment by CharlesDMore** [comment]
I tried to install R using the command "conda install r-base" but was notified that its was already installed. I then updated it using "conda update r-base" and it processed an update and after that the breseq analysis was able to start. I used the test drive sample files and it just finished the analysis and all seems to work fine. It seems the conda installation process installed an updated version of R. 
Thank you for your support!
